File: src-scala-orig/test/scala/cern/colt/matrix/tlong/impl/DenseLargeLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class DenseLargeLongMatrix2DViewTest(arg0: String) extends DenseLargeLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new DenseLargeLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new DenseLargeLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new DenseLargeLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/DenseLargeLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class DenseLargeLongMatrix2DViewTest extends DenseLargeLongMatrix2DTest {

    public DenseLargeLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new DenseLargeLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new DenseLargeLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new DenseLargeLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}
